src/main/java/hudson/scm/IntegrityCheckpointAction.java

```diff
@@ -92,6 +98,19 @@ public static String evalGroovyExpression(Map<String, String> env, String expres
       return result.toString().trim();
     }
   }
+  
+  /**
+	 * @author asen
+	 * Default Whitelist
+	 *
+	 */
+	@Extension
+	public static class DefaultWhitelist extends ProxyWhitelist {
+		public DefaultWhitelist() throws IOException {
+			super(new StaticWhitelist("method java.text.Format format java.lang.Object",
+					"new java.text.SimpleDateFormat java.lang.String"));

```

Comment:
Fine though would be better to just fie a PR for `script-security-plugin`.

Reply:
The PR has been integrated. What is the reason of keeping it?